modules/primer-avatars/lib/avatar-stack.scss

```diff
@@ -43,3 +43,114 @@
     }
   }
 }
+
+// Refactored, new avatar stack:
+
+.AvatarStack {
+  position: relative;
+  min-width: 26px;
+  height: 20px;
+
+  &.AvatarStack--2 {
+    min-width: 36px;
+  }
+
+  &.AvatarStack--3 {
+    min-width: 46px;
+  }
+}
+
+.AvatarStack-body {
```

Comment:
I'm curious to have @broccolini weigh in here about `.AvatarStack` as the containing div that `.AvatarStack-body` is positioned absolutely within.

I spoke to @califa briefly on slack about this and he said that this component will always need to be positioned this way in order to interact with its surroundings as intended. This does, however, go against the principle of "separate container and content" and has the potential to lead to some unnecessary overrides in the future.

Reply:
@sophshep I think this might actually _align_ with that principle.

> Separate container and content: Essentially, this means “rarely use location-dependent styles”. A component should look the same no matter where you put it.

If we treat the container as part of the component as we're doing here, then it _won't_ be location dependent. It'll always appear/behave exactly the same no matter where we put it.

What do you think?

Reply:
I've updated `AvatarStack` to apply `position: absolute;` to `AvatarStack-body` so that it partially addresses mine and @sophshep's concern's (above) re tying the positioning into this component (but not entirely since you have width in the block class). 

@califa - might be good if we have a longer discussion in person about our principles, I'd like to ensure they make sense to everyone and I understand how they might be interpreted differently. We haven't really updated them since we first started the design systems team, and I want to review and iterate on them, and ensure they tie back to [product design principles](https://github.com/github/design/blob/master/docs/design-values-and-principles.md) so that we can use them as a framework to make decisions on both design and code more easily.

To be honest I expect we'll need to iterate on `AvatarStack`(like many things). We want to move to more consistent usage around width and height values so they would point to variables or use utilities, have more layout objects (incl. starting to use CSS grid), and I think we'll have other components that will require this same "stack" treatment than just avatars. But I'm fine with this shipping in the interest of having this for your feature, and it will give us an opportunity to test the new version out.